.github/workflows/slowtests.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Will this run regardless of whether the code repo has changed? I think that would be fine for the weekly tests, but for the daily tests I was hoping that we could check whether the code has changed in the last 24 hours, and avoid running the tests altogether if it hasn't.

Reply:
Right now they are set up to run every day. Are there specific criteria you'd want to use for if the code has changed (eg, changes in the `spacy` directory, or any commits to the branch within the past 24 hours)? There might be a way to do one of those checks as a first step, though it may be a bit hacky :)

Reply:
I would check for any edit, anywhere. If it can't easily be done, or if it would be too hacky, then I would change the frequency of the `develop` and `v4` branches to weekly (for both GPU & slow), and only run the slow tests on `master` daily.

Reply:
Ok, I changed it to something that will (hopefully) check for and only run the tests if there are commits on the relevant branch in the past day... untested for now, but if it looks good we can try it and iterate if needed